src/app.module.ts

```diff
@@ -8,21 +8,36 @@ import { LinksService } from './links/links.service';
 import { AuthModule } from './auth/auth.module';
 import { ConfigModule } from '@nestjs/config';
 import { ScheduleModule } from '@nestjs/schedule';
+import { ThrottlerGuard, ThrottlerModule } from '@nestjs/throttler';
+import { APP_GUARD } from '@nestjs/core';
 
 @Module({
   imports: [
     UsersModule,
     PrismaModule,
     LinksModule,
     AuthModule,
```

Comment:
**suggestion**: Consider using environment variables for `ttl` and `limit` values to make the configuration more flexible.